specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-02/DiskRP/disk.json

```diff
@@ -0,0 +1,856 @@
+          },
+          {
+            "$ref": "#/parameters/DiskNameParameter"
+          },
+          {
+            "$ref": "../common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "If the disk is deleted, this is an expected error code."
+          }
+        },
+        "x-ms-examples": {
+          "Delete a managed disk.": {
+            "$ref": "./examples/diskExamples/Disk_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/disks": {
+      "get": {
+        "tags": [
+          "Disks"
+        ],
+        "operationId": "Disks_ListByResourceGroup",
+        "description": "Lists all the disks under a resource group.",
+        "parameters": [
+          {
+            "$ref": "../common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./diskRPCommon.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DiskList"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all managed disks in a resource group.": {
+            "$ref": "./examples/diskExamples/Disk_ListByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/disks": {
+      "get": {
+        "tags": [
+          "Disks"
+        ],
+        "operationId": "Disks_List",
+        "description": "Lists all the disks under a subscription.",
+        "parameters": [
+          {
+            "$ref": "../common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DiskList"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all managed disks in a subscription.": {
+            "$ref": "./examples/diskExamples/Disk_ListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/disks/{diskName}/beginGetAccess": {
```

Comment:
>beginGetAccess [](http://example.com/codeflow?start=117&length=14)

What does begin - end access does? Are you getting a token? handler? The description is not clear and API kind of explains what it does but doesn't explain access to which entity (e.g. resource group, VM?)